pwnlib/libcdb.py

```diff
@@ -515,6 +517,11 @@ def _find_libc_package_lib_url(libc):
         libc_version = version.group(1).decode()
         yield 'https://launchpad.net/ubuntu/+archive/primary/+files/libc6_{}_{}.deb'.format(libc_version, libc.arch)
 
+    # check debian.sipwise.com if it's a debian libc
+    maybe_deb_version = maybe_deb_version.split('/')[-1]
```

Comment:
This throws AttributeError if maybe_deb_version is None, please move under the if below.